Engine/Module/ModuleEditor.cpp

```diff
@@ -91,6 +92,10 @@ bool ModuleEditor::InitImgui()
 
 update_status ModuleEditor::PreUpdate()
 {
+#if GAME
```

Comment:
Same comment as brefore.

Reply:
In this case editor is needed in some classes for naming GameObjects, using the hierarchy info stored in it.